ev3dev/helper.py

```diff
@@ -146,6 +147,217 @@ class MediumMotor(ev3dev.auto.MediumMotor, MotorMixin):
+        motors = motors if motors is not None else self.motors.values()
+
+        for motor in motors:
+            motor.reset()
+
+    def stop(self, motors=None):
+        motors = motors if motors is not None else self.motors.values()
+
+        for motor in motors:
+            motor.stop()
+
+    def _is_state(self, motors, state):
+        motors = motors if motors is not None else self.motors.values()
+
+        for motor in motors:
+            if state not in motor.state:
+                return False
+
+        return True
+
+    @property
+    def is_running(self, motors=None):
+        return self._is_state(motors, ev3dev.auto.LargeMotor.STATE_RUNNING)
+
+    @property
+    def is_ramping(self, motors=None):
+        return self._is_state(motors, ev3dev.auto.LargeMotor.STATE_RAMPING)
+
+    @property
+    def is_holding(self, motors=None):
+        return self._is_state(motors, ev3dev.auto.LargeMotor.STATE_HOLDING)
+
+    @property
+    def is_overloaded(self, motors=None):
+        return self._is_state(motors, ev3dev.auto.LargeMotor.STATE_OVERLOADED)
+
+    @property
+    def is_stalled(self):
+        return self._is_state(motors, ev3dev.auto.LargeMotor.STATE_STALLED)
+
+    def wait(self, cond, timeout=None, motors=None):
+        motors = motors if motors is not None else self.motors.values()
+
+        for motor in motors:
+            motor.wait(cond, timeout)
+
+    def wait_until_not_moving(self, timeout=None, motors=None):
+        motors = motors if motors is not None else self.motors.values()
+
+        for motor in motors:
+            motor.wait_until_not_moving(timeout)
+
+    def wait_until(self, s, timeout=None, motors=None):
+        motors = motors if motors is not None else self.motors.values()
+
+        for motor in motors:
+            motor.wait_until(s, timeout)
+
+    def wait_while(self, s, timeout=None, motors=None):
+        motors = motors if motors is not None else self.motors.values()
+
+        for motor in motors:
+            motor.wait_while(s, timeout)
+
+
+class MotorPair(MotorSet):
+
+    def __init__(self, motor_specs, desc=None):
+        MotorSet.__init__(self, motor_specs, desc)
+        (self.left_motor, self.right_motor) = self.motors.values()
+        self.max_speed = self.left_motor.max_speed
+
+    def set_speed_ratio(self, left_motor_ratio, right_motor_ratio, set_right=True):
+        """
+        Set the speeds of the left_motor vs right_motor by ratio
+        """
+
+        # Set the right_motor speed relative to the left_motor_speed
+        if set_right:
+            self.right_motor.speed_sp = int((self.left_motor.speed_sp * left_motor_ratio) / right_motor_ratio)
+        else:
+            self.left_motor.speed_sp = int((self.right_motor.speed_sp * right_motor_ratio) / left_motor_ratio)
+
+    def set_speed_percentage(self, left_motor_percentage, right_motor_percentage):
+        """
+        Set the speeds of the left_motor vs right_motor by percentage of their maximum speed
```

Comment:
We should specify that the units are -100 to 100. The term "percentage" is often  used to refer to [-1,1] or [0,1] by some programmers.

Reply:
Ack...will also add some asserts to verify the inputs are within the range